.github/workflows/aws_tfhe_fast_tests.yml

```diff
@@ -12,6 +12,7 @@ env:
   SLACK_USERNAME: ${{ secrets.BOT_USERNAME }}
   SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
   IS_PULL_REQUEST: ${{ github.event_name == 'pull_request' }}
+  NODE_VERSION: 22.6
```

Comment:
use makefile version ? if needed add a command in the makefile to print out that version

Reply:
and remove this line